ActivityView.m

```diff
@@ -122,6 +123,17 @@ - (NSArray*)excludedActivitiesForKeys:(NSArray*)passedKeys {
             activityView.popoverPresentationController.permittedArrowDirections = 0;
         }
     }
+    
+    [activityView setCompletionWithItemsHandler:
+        ^(NSString *activityType, BOOL completed, NSArray *returnedItems, NSError *activityError) {
+            NSDictionary *payload = @{
+                @"activityType": RCTNullIfNil(activityType),
+                @"completed": @(completed),
+                @"returnedItems": RCTNullIfNil(returnedItems)
+            };
+            callback(@[RCTJSErrorFromNSError(activityError), payload]);
```

Comment:
Looks like we have to do an if here and only call RCTJS.... when activityError != null, otherwise it's `UnspecifiedNativeError`